modules/raft/src/main/java/org/apache/ignite/raft/jraft/disruptor/StripedDisruptor.java

```diff
@@ -214,7 +214,7 @@ void unsubscribe(NodeId nodeId) {
             assert handler != null : format("Group of the event is unsupported [nodeId={}, event={}]", event.nodeId(), event);
 
             //TODO: IGNITE-15568 endOfBatch should be set to true to prevent caching tasks until IGNITE-15568 has fixed.
-            handler.onEvent(event, sequence, true);
+            handler.onEvent(event, sequence, subscribers.size() > 1 ? true : endOfBatch);
```

Comment:
Is it an intended change?

Reply:
Not intended, but I decided to leave it after commit. Because this allows to do slightly better of disruptor for Meta storage and Cluster management group and I don't want to create dedicate ticket for this small change.

Reply:
Sounds reasonable, however I had to ask. How did you understand that it works better after given change?

Reply:
We don't have a test that will be able to demonstrate that. I assume, that update is not doing worse at least.
Theoretically, RAFT log may fsync less often in the circumstance when several RAFT commands are executing simultaniously.

Reply:
I don't think that this change is reasonable, because we have a plan to port disruptor changes in a upcoming sprint, potentially this fix will live a few weeks

Reply:
We will work a little better just now. Today, the change worth nothing for us.